Week1/homework/app.js

```diff
@@ -1,11 +1,143 @@
 'use strict';
-
 {
-  const bookTitles = [
-    // Replace with your own book titles
-    'harry_potter_chamber_secrets',
-  ];
+  //the_brothers_karamazov
+  //three_men_in_a_boat
+  //ulysses
+  //the_great_gatsby
+  //war_and_peace
+  //the_odyssey
+  //crime_and_punishment
+  //the_stranger
+  //the_sun_also_rises
+  //candide
+
+  // 1.1
+  // const bookList = [
+  //   'the_brothers_karamazov ',
+  //   'three_men_in_a_boat',
+  //   'ulysses',
+  //   'the_great_gatsby ',
+  //   'war_and_peace',
+  //   'the_odyssey',
+  //   'crime_and_punishment',
+  //   'the_stranger ',
+  //   'the_sun_also_rises',
+  //   'candide'
+  // ];
+
+  //1.3
+  // function general() {
+  //   const newList = document.createElement('ul');
+
+  //   for (let i = 0; i < bookList.length; i++) {
+  //     const book = bookList[i];
+  //     newList.innerHTML += `<li>${book}</li>`;
+  //   }
+
+  //   return newList;
+  // }
+
+  // document.body.append(general());
+
+  //1.4
+  const bookList = {
+    the_brothers_karamazov: {
+      title: 'The Brothers Karamazov',
+      author: 'Fyodor Dostoyevski',
+      language: 'English',
+    },
+    three_men_in_a_boat: {
+      title: 'Three Men In a Boat',
+      author: 'Jerome Klapka Jerome',
+      language: 'English',
+    },
+    ulysses: {
+      title: 'Ulysses',
+      author: 'James Joyce',
+      language: 'English',
+    },
+    the_great_gatsby: {
+      title: 'The Great Gatsby',
+      author: 'F. Scott Fitzgerald',
+      language: 'English',
+    },
+    war_and_peace: {
+      title: 'War and Peace',
+      author: 'Lev Tolstoy ',
+      language: ' ‎Russian',
+    },
+    the_odyssey: {
+      title: 'The Odyssey',
+      author: 'Homeros ',
+      language: ' Greek',
+    },
+    crime_and_punishment: {
+      title: 'Crime and Punishment',
+      author: 'Fyodor Dostoyevski ',
+      language: ' ‎Russian',
+    },
+    the_stranger: {
+      title: 'The Stranger',
+      author: 'Albert Camus',
+      language: 'French',
+    },
+    the_sun_also_rises: {
+      title: 'The Sun Also Rises',
+      author: 'Ernest Hemingway',
+      language: 'English',
+    },
+    candide: {
+      title: 'Candide',
+      author: 'Voltaire',
+      language: 'French',
+    },
+  };
+  //1.7
+
+  const covers = {
+    the_brothers_karamazov: 'images/the-brothers-karamazov-7.jpg',
+    three_men_in_a_boat: 'images/three_men_in_a_boat.jpg',
+    ulysses: 'images/ulysses.jpg',
+    the_great_gatsby: 'images/the_great_gatsby.jpg',
+    war_and_peace: 'images/war_and_peace.jpg',
+    the_odyssey: 'images/the_odyssey.jpg',
+    crime_and_punishment: 'images/crime_and_punishment.jpg',
+    the_stranger: 'images/the_stranger.jpg',
+    the_sun_also_rises: 'images/the_sun_also_rises.jpg',
+    candide: 'images/candide.jpg',
+  };
+
+  const section = document.createElement('section');
+  document.body.prepend(section);
```

Comment:
You should use `appendChild()`
```suggestion
  document.body.appendChild(section);
```

Reply:
Yes you right I could be used appendChild but If you have a much JavaScript, it can visibly slow the loading of your page because it loads all of the JavaScript before it loads any of the HTML.
When you place your JavaScript links at the bottom of your HTML body, it gives the HTML time to load before any of the JavaScript loads, which can prevent errors, and speed up website response time.